framework/h5py_interface_creator.py

```diff
@@ -29,13 +29,20 @@
 from utils import utils, mathUtils
 from BaseClasses import InputDataUser, MessageUser
 
-def _dumps(val):
+# the database version should be modified
+# everytime a new modification of the internal
+# structure of the data is performed
+_hdf5DatabaseVersion = "v1.0"
```

Comment:
any way we can pin these to RAVEN releases instead of a separate release schedule for this sub-part? Since there's no mechanic to force this version update, chances of it being overlooked are very high.

Reply:
Agree 100%... I didn't find a simple way to link the "Version" to the hash or to the tag because I did not want to assume that git is present (maybe they downloaded with zip, for example) or that the tag version (e.g. RAVENv2.1) is the version of this structure or that any commit on this file would bump the version (e.g. if I modify a line that does not modify the internal structure, it should not require for a conversion).... Basically I have not a good idea on how to do it... any idea is welcome (because honestly I do not have a good one ehehe)